go-controller/pkg/util/ovn.go

```diff
@@ -48,3 +49,21 @@ func PlatformTypeIsEgressIPCloudProvider() bool {
 		config.Kubernetes.PlatformType == string(ocpconfigapi.AzurePlatformType) ||
 		config.Kubernetes.PlatformType == string(ocpconfigapi.OpenStackPlatformType)
 }
+
+// Starting with v21.03.0 OVN sets OVS.Interface.external-id:ovn-installed
+// and OVNSB.Port_Binding.up when all OVS flows associated to a
+// logical port have been successfully programmed.
+// OVS.Interface.external-id:ovn-installed can only be used correctly
+// in a combination with OVS.Interface.external-id:iface-id-ver
+func GetOVNIfUpCheckMode() (bool, error) {
+	if _, stderr, err := RunOVNSbctl("--columns=up", "list", "Port_Binding"); err != nil {
+		if strings.Contains(stderr, "does not contain a column") {
+			klog.Infof("Falling back to using legacy OVS flow readiness checks")
+			return false, nil
+		}
+		return false, fmt.Errorf("failed to check if port_binding is supported in OVN, stderr: %q, error: %v",
+			stderr, err)
+	}
+	klog.Infof("Detected support for port binding with external IDs")
```

Comment:
For each network (default and secondary), this will need to be called twice: The first time will be called by network manager when ovnkube-node is first started, which indicate the initial value, and secondary time will be called by individure network controller, waiting for the ovnkube-master's upgrade finishes and see if there is any change of this 
"OVNIfUp" suppport, if yes, updating the atomicOvnUpEnabled field used by watchPodsDPU()